addons/delivery/models/sale_order.py

```diff
@@ -85,8 +85,10 @@ def action_open_delivery_wizard(self):
 
     def _create_delivery_line(self, carrier, price_unit):
         SaleOrderLine = self.env['sale.order.line']
+        context = {}
         if self.partner_id:
             # set delivery detail in the customer language
+            context['lang'] = self.partner_id.lang
```

Comment:
I think it's worth adding a comment to explain why this "unused" variable is actually useful for the translation process, with some details or even a redirection to the relevant code
https://github.com/odoo/odoo/blob/cff91c6acbb7ef6a884d4dc4170574b83ff6e4cb/odoo/tools/translate.py#L416-L417